Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ruby:2.3-slim
+MAINTAINER OpenStreetMap
+ENV REFRESHED_AT 2016-09-15
+
+# Install packages
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  build-essential \
+  ruby-dev \
+  libgdbm-dev \
+  libncurses5-dev \
+  libffi-dev \
+  libyaml-dev \
+  libreadline-dev \
+  libpq-dev \
+  libxml2-dev \
+  imagemagick \
+  libmagickwand-dev \
+  postgresql-client \
+  nodejs \
+  file
+
+# Setup app location
+RUN mkdir -p /app
+WORKDIR /app
+
+# Install gems
+ADD Gemfile /app/Gemfile
+ADD Gemfile.lock /app/Gemfile.lock
```

Comment:
These are required so that bundler can install all the gem dependencies required for the app. They have to be manually added so that they are available during the image build process.